reference/platforms.md

```diff
@@ -10,9 +10,9 @@
 | DAM        | Digital Asset Management              |
 | MA         | Marketing Automation                  |
 | POS        | Point of Sale                         |
-| Search     | and Findability                       |
+| Search     | Search and Findability                |
 | Recommendation     | Content or product recommendation                       |
-| Customer   | Service                               |
+| Enterprise Help Desk   | Customer Service Platform |
```

Comment:
I think this table is meant to explain, among other things, this diagram's terminology: https://netlify.slack.com/archives/D06H8J2NBNF/p1717086989018409

I don't see "Customer" or "Enterprise Help Desk" but I do see "Customer Service". Perhaps this row should be "Customer Service" on the left side and an explanation on the right side?

Reply:
How about the suggested change? I was struggling with this as it does not seem to have a widely accepted acronym, so a bit weird to "resolve" this...